.circleci/config.yml

```diff
@@ -4,33 +4,10 @@ orbs:
   gcp-gke: circleci/gcp-gke@1.4.0
   go: circleci/go@1.7.1
   codecov: codecov/codecov@1.1.0
-
-trigger:
-  tags:
-    include:
-      - '*'
-  branches:
-    include:
-      - "master"
-      - "release/*"
-      - "refs/tags/*"
-
-pr:
-  - "master"
-  - "release/*"
```

Comment:
`.github/workflows/ci-circleci.yml` is what triggers this job now, so this shouldn't be required